app/views/recipe_categories/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Recipe categories
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= RecipeCategory.count %> recipe categories!
```

Comment:
Avoid doing queries in your views!
Move this to the controller. Also, take a look at [this article](https://medium.com/ruby-daily/comparing-count-length-and-size-in-ruby-on-rails-e061840bc19e) (Actually I was convinced `size` was always the way to go, but It seems that doesn't free us from doing queries to the DB in some scenarios, I personally avoid using count)

Actually you already have  `@recipe_categories` in memory 👀

Reply:
Thanks a bunch for your feedback, Luis. Already fixed in the latest commit!